### ISSUE TITLE
Support external Bluetooth logger

### ISSUE THREAD

**Issue by boergmar** [nguillaumin/osmtracker-android#77]
Have greater reception accuracy than internal smartphone GPS

**Comment by nguillaumin** [comment]
Yeah I think that was already requested a long time ago when the project was hosted at Google Code (and I wasn't able to transfer the issues to GitHub unfortunately).

I think the problem at the time was that the Android SDK didn't have good APIs to interact with bluetooth receivers but that may have changed... Although that would mean dropping support for older versions of Android.

**Comment by Binnette** [comment]
I think using the "Mock location" service on the phone throught Developper options can be a nice workaround.
And so we can find some time to implements this later or find another way to do it.
